engine/config.go

```diff
@@ -270,9 +270,10 @@ type OgcAPIStyles struct {
 }
 
 type OgcAPIFeatures struct {
-	Limit       Limit                 `yaml:"limit"`
-	Datasources *Datasources          `yaml:"datasources"`
-	Collections GeoSpatialCollections `yaml:"collections" validate:"required"`
+	Limit         Limit                 `yaml:"limit"`
+	Datasources   *Datasources          `yaml:"datasources"`
+	BackgroundMap string                `yaml:"backgroundMap"`
```

Comment:
prima zo, zou alleen `backgroundMap` naar `basemap` veranderen. Dat is wel echt de geoterm die gebruikt wordt voor achtergrondkaarten: https://gisgeography.com/basemap/

Reply:
done